app/(static)/deck/components/DropdownProto.tsx

```diff
@@ -1,6 +1,8 @@
-import { Menu, Transition } from "@headlessui/react";
+import { Button } from "@/components/ui/button";
 import { CheckIcon, ChevronDownIcon, ChevronUpIcon } from "lucide-react";
-import { Fragment, Dispatch, SetStateAction } from "react";
+import { Dispatch, SetStateAction, useState } from "react";
+import { PopoverContent } from "@radix-ui/react-popover";
+import { PopoverTrigger, Popover } from "@/components/ui/popover";
 
 function classNames(...classes: string[]) {
```

Comment:
If you want you can replace this with `cn` from `@/lib/utils`. it's a better way to merge tailwind classes then this helper function.

Reply:
there is same code written in two different files 
DropdownProto.tsx -> app/(static)/deck/components/DropdownProto.tsx
dropdownpropt.tsx -> components/web/alternatives/dropdownproto.tsx

should i make the same changes as removing the headlessui components with shadcn in both 
or remove any one file and make the corresponding file import changes related to other